tests/unit/completion/test_models.py

```diff
@@ -269,29 +273,31 @@ def test_url_completion(qtmodeltester, config_stub, web_history, quickmarks,
 
     Verify that:
         - quickmarks, bookmarks, and urls are included
-        - no more than 'web-history-max-items' history entries are included
+        - no more than 'web-history-max-items' items are included (TODO)
```

Comment:
I don't really know - I can't see how it's a problem to have more history to filter. If you're not interested in it it's scrolled down and hidden anyways, but if you are, you can search for things there.

If anything, it might make sense to have some (rather high) cutoff for on-disk history, to keep the sqlite database smaller. But I don't think we need this currently.

I'm all for ditching it and seeing if someone complains.